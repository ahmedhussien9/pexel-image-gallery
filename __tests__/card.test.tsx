
import { render, screen } from '@testing-library/react'
import Card from '../components/CardContainer/CardConainer'

describe('Card', () => {
    // it('renders Card Component', () => {
    //     const { container } = render(
    //         <Card>
    //             <h1>Hey</h1>
    //         </Card>
    //     )
    //     console.log(container)
    //     expect(container).toMatchSnapshot()
    // });

})