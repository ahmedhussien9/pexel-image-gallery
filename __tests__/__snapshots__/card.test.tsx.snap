// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders Card Component 1`] = `
<div>
  <div
    class="card"
  >
    <h1>
      Hey
    </h1>
  </div>
</div>
`;

exports[`Card renders Photo gallery page 1`] = `
<div>
  <div
    class="card"
  >
    <h1>
      Hey
    </h1>
  </div>
</div>
`;
